### PR TITLE
Feature-IOPS-2377-WGS-Test-Order-Form-Direct-to-LAB 

### PR DESCRIPTION
**Pull request by KazeemHamzat**
Added Provenance to show history of SR, and when sample was provided.

### CODE REVIEW

File: ServiceRequest/ServiceRequest-WGSTestOrderFormUpdated-DirectToLab-Example.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
   ],
   "relevantHistory": [
     {
-      "reference": "Provenance/Provenance-ServiceRequest-WGSTestOrderForm-DirectToLab-Example"
+      "reference": "Provenance/Provenance-Provenance-WGSTestOrderForm-DirectToLab-Example"
```
Comment (omarisgreat) [inline ServiceRequest/ServiceRequest-WGSTestOrderFormUpdated-DirectToLab-Example.json:125]: Duplicated 'Provenance' in name

**Review by omarisgreat — APPROVED** [review]
Small issue with duplicated name but can be merged without re-review once fixed

File: ServiceRequest/ServiceRequest-WGSTestOrderFormUpdated-DirectToLab-Example.json (hunk 1 above)
Comment (KazeemHamzat) [reply ServiceRequest/ServiceRequest-WGSTestOrderFormUpdated-DirectToLab-Example.json:125]: Not a duplicate but a name change to Provenance-Provenance-WGSTestOrderForm-DirectToLab-Example

Comment (omarisgreat) [reply ServiceRequest/ServiceRequest-WGSTestOrderFormUpdated-DirectToLab-Example.json:125]: Hi Kazeem, on the files changed, the filename is Provenance/Provenance-WGSTestOrderForm-DirectToLab-Example.json (1 Provenance in the path, and 1 in the file name), the reference has 3

Comment (KazeemHamzat) [reply ServiceRequest/ServiceRequest-WGSTestOrderFormUpdated-DirectToLab-Example.json]: Now sorted as expected.

**Pull request merged**